patches/mink-361-selenium-driver-switch-to-iframe-1.patch

```diff
@@ -0,0 +1,79 @@
+From 3d6e0e22a3a43477c7ea9e5d6dcb2e62777c3dc8 Mon Sep 17 00:00:00 2001
+From: Omar <omar.lopesino@metadrop.net>
+Date: Tue, 14 Feb 2023 17:23:51 +0100
+Subject: [PATCH 1/3] Iframe switch - Send element info as new selenium
+ versions require it
+
+---
+ src/Selenium2Driver.php | 12 +++++++++++-
+ 1 file changed, 11 insertions(+), 1 deletion(-)
+
+diff --git a/src/Selenium2Driver.php b/src/Selenium2Driver.php
+index ee7f2e75..4f5cc759 100755
+--- a/src/Selenium2Driver.php
++++ b/src/Selenium2Driver.php
+@@ -423,7 +423,17 @@ public function switchToWindow($name = null)
+      */
+     public function switchToIFrame($name = null)
+     {
+-        $this->wdSession->frame(array('id' => $name));
++        if (!empty($name)) {
++          $element = $this->wdSession->element('id', $name);
++          $element_info = [
++            'ELEMENT' => $element->getID(),
++          ];
++        }
++        else {
++          $element_info = NULL;
++        }
++
++        $this->wdSession->frame(array('id' => $element_info));
+     }
+ 
+     /**
+
+From 97ed86ab48de4cd9da39d0dc1d2b70249456cf99 Mon Sep 17 00:00:00 2001
+From: Omar <omar.lopesino@metadrop.net>
+Date: Tue, 14 Feb 2023 17:33:08 +0100
+Subject: [PATCH 2/3] Iframe switch - Use lowercase null constant
+
+---
+ src/Selenium2Driver.php | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/Selenium2Driver.php b/src/Selenium2Driver.php
+index 4f5cc759..b48bacc0 100755
+--- a/src/Selenium2Driver.php
++++ b/src/Selenium2Driver.php
+@@ -430,7 +430,7 @@ public function switchToIFrame($name = null)
+           ];
+         }
+         else {
+-          $element_info = NULL;
++          $element_info = null;
```

Comment:
@mistermoper why null are lowercase in some sites and uppercase in other places like https://github.com/Metadrop/drupal-boilerplate/pull/106/files#diff-150ac77db3caab46bb2132ed3502198eb1fe25b29b9b82c3738dfec636916d47R27